highlight.io/components/Competitors/hotjar.ts

```diff
@@ -1,37 +1,47 @@
 import { Competitor } from './competitors'
+import {
+	EngineersVsMarketers,
+	HighlightIsOpenAndTransparent,
+	HighlightShipsNewFeatures,
+} from './shared-snippets'
 
 export const HotjarSpec: Competitor = {
 	name: 'Hotjar',
 	header: 'The Open Source Hotjar Alternative.',
 	subheader:
 		'Pixel-perfect video replay of your frontend web application. Step into the shoes of your users.',
-	type: 'session-replay',
 	sections: [
 		{
 			title: 'General',
 			rows: [
 				{
 					feature: 'Unlimited Team Members',
+					tooltip:
+						'LogRocket only allows 5 team members on their free plan.',
 					highlight: 1,
 					competitor: 0,
```

Comment:
nit: numbers here could be booleans if that is their only purpose

Reply:
For competitor, the only option is 0 or 1, but for highlight, the options are 0, 0.5, 1. 0.5 will represents "coming soon". For consistency, I opted to keep the numbers for the competitor.